python/adxl345_helper.py

```
@@ -0,0 +1,16 @@
+import time
```

Ruff (0.8.2)

`time` imported but unused

Remove unused import: `time`

(F401)